Rmds/star.Rmd

```diff
@@ -63,8 +71,10 @@
       --genomeLoad NoSharedMemory
     ```
 
-## Resources
+## Group assignment
+1. Which samples from the paper did you choose, and what did you want to test with them?
+2. Create a script that performs the STAR alignment for your files, avoiding excessive repetition, and can be run in a way that it is not interrupted when you close your connection with the server.
```

Comment:
```suggestion
## Group assignment

1. Which samples from the paper did you choose, and what did you want to test with them?
2. Create a script that performs the STAR alignment for your files, avoiding excessive repetition, and can be run so that it is not interrupted when you close your connection with the server.
```